ec2utils/ec2deprecateimg/lib/ec2utils/ec2deprecateimg.py

```diff
@@ -73,30 +73,14 @@ def __init__(
     # ---------------------------------------------------------------------
     def _find_images_by_id(self, image_id, filter_replacement_image=None):
```

Comment:
Because for deprecation we have the special condition that one can use the virtualization type as an additional filter. Somehow we need to trim the list of potential images to be deprecated based on this condition.